src/tests/sqlsmith/src/sql_gen/agg.rs

```diff
@@ -39,7 +39,12 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
             .map(|t| self.gen_expr(t, context))
             .collect();
 
-        let distinct = self.flip_coin() && self.is_distinct_allowed && !exprs.is_empty();
+        // see `Binder::bind_normal_agg`
+        let distinct_allowed = func.func != AggKind::ApproxCountDistinct
+            && !exprs.is_empty()
+            && exprs.iter().skip(1).all(|e| matches!(e, Expr::Value(_)));
```

Comment:
Why `skip(1)` here?

Reply:
Because with current distinct dedup implementation, only the first argument can be non-const expression. And in binder we just restrict arguments from the second one to the last one to be literal value.

Reply:
Could you please leave this comment inline inside the code? Thanks!